certora/specs/SocialRecoveryModule.spec

```diff
@@ -169,7 +170,7 @@ rule guardianCanAlwaysBeRevoked(env e, address guardian, address prevGuardian, u
     require guardianStorageContract.countGuardians(safeContract) > threshold;
     // The address should be a guardian.
     require currentContract.isGuardian(safeContract, guardian);
-    require guardianStorageContract.entries[safeContract].guardians[guardian] != guardian;
+    require guardianStorageContract.isGuardianInGuardianList(safeContract, guardian);
```

Comment:
To me, this belongs in `requireGuardiansLinkedListIntegrity`, since it is something that is guaranteed by the linked list integrity.

Reply:
Moved 👍🏾